From 706e89571b2e80c12e7fac4f6bcdc913e5de6636 Mon Sep 17 00:00:00 2001
From: jacaru <1446775+jacaru@users.noreply.github.com>
Date: Sat, 26 Aug 2017 14:32:59 +0200
Subject: [PATCH 01/13] Add utcnow to template entry rendering

---
 flexget/utils/template.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/flexget/utils/template.py b/flexget/utils/template.py
index 26cd25aa3..1427a89ac 100644
--- a/flexget/utils/template.py
+++ b/flexget/utils/template.py
@@ -271,6 +271,7 @@ def render_from_entry(
     # Make a copy of the Entry so we can add some more fields
     variables = copy(entry.store)
     variables['now'] = datetime.now()
+    variables['utcnow'] = datetime.utcnow()
     # Add task name to variables, usually it's there because metainfo_task plugin, but not always
     if hasattr(entry, 'task') and entry.task is not None:
         if 'task' not in variables:
-- 
2.20.1

