From 3d5133a5a18bf9756bdc4f3d4db6e7cfff80c47d Mon Sep 17 00:00:00 2001
From: Your Name <1446775+jacaru@users.noreply.github.com>
Date: Mon, 8 Jan 2024 22:40:41 +0100
Subject: [PATCH 13/13] Log non-matched conditions

---
 flexget/plugins/filter/if_condition.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/flexget/plugins/filter/if_condition.py b/flexget/plugins/filter/if_condition.py
index 392561c7a..2c1a80e77 100644
--- a/flexget/plugins/filter/if_condition.py
+++ b/flexget/plugins/filter/if_condition.py
@@ -43,6 +43,8 @@ class FilterIf:
             passed = evaluate_expression(condition, eval_locals)
             if passed:
                 logger.debug('{} matched requirement {}', entry['title'], condition)
+            else:
+                logger.trace('{} did not match requirement {}', entry['title'], condition)
         except UndefinedError as e:
             # Extract the name that did not exist
             missing_field = e.args[0].split("'")[1]
-- 
2.30.2

